lockfile.json

```diff
@@ -213,7 +213,7 @@
     "revision": "8f6b1f357d1231c420404b5f7a368a73c25adfa2"
   },
   "vim": {
-    "revision": "59595443fb486449f42db839934117221358a85f"
+    "revision": "7d883fb13bf4f9038522c9d8b3197b821498ab1d"
```

Comment:
I think CI is failing because this commit doesn't exist. https://github.com/vigoux/tree-sitter-viml/commits/master